lib/src/builder/codegen/component_generator.dart

```diff
@@ -225,5 +219,8 @@ class _LegacyComponentGenerator extends ComponentGenerator {
 
   @override
   String get defaultConsumedPropsImpl =>
-      'final List<ConsumedProps> \$defaultConsumedProps = const [${propsNames.metaConstantName}]';
+      '  /// The default consumed props, taken from ${propsNames.consumerName}.\n'
+      '  /// Used in -`ConsumedProps` methods if [consumedProps] is not overridden.\n'
```

Comment:
```suggestion
      '  /// Used in `ConsumedProps` methods if [consumedProps] is not overridden.\n'
```

Reply:
Addressed in e3f27c9c597555e0e3cb917a34687c9b9859fd57